view/frontend/web/js/view/payment/method-renderer/omise-cc-googlepay-method.js

```diff
@@ -125,6 +125,13 @@ define(
 
             code: 'omise_cc_googlepay',
 
+            logo: {
+                file: "images/googlepay.svg",
```

Comment:
@aashishgurung @ajzkk why do we name differently ("googlepay.svg") than in https://github.com/omise/omise-woocommerce/pull/368/files ("google-pay.svg")?

Reply:
I will update this on wooCommerce.
Since `grabpay` and `shopeepay` are also without dash.